### DISCUSSION TITLE
Why is the property "Charts" commented on IXLWorksheet.cs?

### DISCUSSION THREAD

**Discussion by TooMuchLagWillKillYou** [ClosedXML/ClosedXML#2355] [Q&A]
Hello!
I've been working with ClosedXML in the last month, what a wonderful library!

I was trying to find a way to get the charts in an Excel template, and I found out that there is a "Charts" property of type `IXLCharts` in `IXLWorksheet.cs` but it is commented. Can I ask why? Is it because charts are not yet fully implemented in ClosedXML?

Thank you so much for your attention.

**Comment by igitur** [comment]
Charts are not implemented. We inherited the codebase with that commented code. I don't know why the previous author included it.

**Reply by TooMuchLagWillKillYou** [reply]
Can I ask if you're planning to implement charts in the future?

**Comment by jahav** [comment]
At some point yes. But that's at least a year (nearly certainly more). Do not make any plans counting on that
